Data/ByteString/Lazy/Internal.hs

```diff
@@ -204,9 +204,8 @@ foldrChunks f z = go
 -- accumulating left fold.
 foldlChunks :: (a -> S.ByteString -> a) -> a -> ByteString -> a
 foldlChunks f = go
-  where go a _ | a `seq` False = undefined
```

Comment:
I had accidentally come across this weird bit, and ended up looking for more `seq` weirdness. The other uses of `seq` are more benign though.